.github/workflows/nucliadb_standalone.yml

```diff
@@ -75,8 +75,10 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      max-parallel: 2
       matrix:
         python-version: ["3.12"]
+        storage_backend: ["gcs", "s3", "local"]
```

Comment:
Coming back, since we are using lazy fixtures, why not just do a single run of this with `TESTING_STORAGE_BACKEND=gcs,s3,local`? I guess it would be too slow since most tests depend on this fixture.

Anyway, feel free to disregard all this, mainly thinking out loud about CI.

Reply:
I actually agree with you, it's overkill in this case.
Plus I am realizing now that some of the tests are written assuming local storage only, so I would need to refactor them too...

I will parametrize the upload/download test instead 👍🏼